### PR TITLE
Add `osu! South East Asia Tournament 5` wiki article

### PR DESCRIPTION
**Pull request by Nivalyx**
wiki page for the currently ongoing [osu! South East Asia Tournament 5](http://osu.ppy.sh/community/forums/topics/1452561?n=1) (it's that time of the year again, yup)

please ignore the duplicate maps in the mappool btw; those aren't a mistake as the mappool is... *specifically intended* to be that way 🐇 

---

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/oSEAT/5/en.md [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+tags:
+  - oSEAT
+  - oSEAT5
+---
+
+# osu! South East Asia Tournament 5
+
+![oSEAT5 logo](img/logo.jpg)
+
+The **osu! South East Asia Tournament 5** (***oSEAT5***) is a double-elimination 1v1 osu! tournament hosted by ![][flag_MY] [Its not Marvin](https://osu.ppy.sh/users/13600612), ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), and ![][flag_FR] [Taevas](https://osu.ppy.sh/users/7276846). The tournament is open to all players from all ten [ASEAN member states](https://asean.org/asean/asean-member-states/) (![][flag_BN] Brunei Darussalam, ![][flag_KH] Cambodia, ![][flag_ID] Indonesia, ![][flag_LA] Laos, ![][flag_MY] Malaysia, ![][flag_MM] Myanmar, ![][flag_PH] The Phillipines, ![][flag_SG] Singapore, ![][flag_TH] Thailand, and ![][flag_VN] Vietnam) regardless of rank. It is the 5th instalment of the osu! South East Asia Tournament.
```
Comment (Walavouchey) [inline wiki/Tournaments/oSEAT/5/en.md:11]: dead link, probably moved

```suggestion
The **osu! South East Asia Tournament 5** (***oSEAT5***) is a double-elimination 1v1 osu! tournament hosted by ![][flag_MY] [Its not Marvin](https://osu.ppy.sh/users/13600612), ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), and ![][flag_FR] [Taevas](https://osu.ppy.sh/users/7276846). The tournament is open to all players from all ten [ASEAN member states](https://asean.org/about-asean/member-states/) (![][flag_BN] Brunei Darussalam, ![][flag_KH] Cambodia, ![][flag_ID] Indonesia, ![][flag_LA] Laos, ![][flag_MY] Malaysia, ![][flag_MM] Myanmar, ![][flag_PH] The Phillipines, ![][flag_SG] Singapore, ![][flag_TH] Thailand, and ![][flag_VN] Vietnam) regardless of rank. It is the 5th instalment of the osu! South East Asia Tournament.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/oSEAT/5/en.md [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+   - Players who ranked **#17 - #24** based on their accumulated Qualifier Points will qualify to the **upper Preliminary group bracket** as **Seed C** players.  
+   - Players who ranked **#25 - #32** based on their accumulated Qualifier Points will qualify to the **upper Preliminary group bracket** as **Seed D** players.    
+   - Players who ranked **#33 - #48** based on their accumulated Qualifier Points will qualify to the **lower Preliminary group bracket** as **Seed A** players.   
+   - Players who ranked **#49 - #64** based on their accumulated Qualifier Points will qualify to the **lower Preliminary group bracket** as **Seed B** players. 
+   - Players who ranked **#65 - #80** based on their accumulated Qualifier Points will qualify to the **lower Preliminary group bracket** as **Seed C** players. 
+   - Players who ranked **#81 - #96** based on their accumulated Qualifier Points will qualify to the **lower Preliminary group bracket** as **Seed D** players. 
+   - Players who ranked #97 or lower based on their accumulated Qualifier Points will be knocked out from the tournament.
```
Comment (Walavouchey) [inline wiki/Tournaments/oSEAT/5/en.md:383]: table

File: wiki/Tournaments/oSEAT/5/en.md [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+3. The modified Double Elimination System works as follows:
+   - Players who lose in the Upper Bracket can still play again on the Lower Bracket.
+   - Players who lose in the Lower Bracket will be eliminated from the tournament.
+   - In the Grand Final, the winner of the the Upper Bracket will start the match with a 2-point advantage against the winner of the Lower Bracket. **There will be no Grand Final bracket reset.**
+4. Players who can compete in the next round are determined by the following criteria:
+   - In the Round of 64, Round of 32, and Round of 16, each player needs to win 6 points in order to win a match. (Best-of-11)
+   - In the Quarterfinals and Semifinals, each player needs to win 7 points in order to to win a match. (Best-of-13)
+   - In both of the Finals weeks (excluding the Grand Final), each player needs to win 8 points in order to to win a match. (Best-of-15)   
+   - In the Grand Final, each player needs to win 9 points in order to to win a match (Best-of-17) with the winner of the the Upper Bracket starting the match with a 2-point advantage as described above.
+   - Players who Win By Default.
```
Comment (Walavouchey) [inline wiki/Tournaments/oSEAT/5/en.md:414]:
```suggestion
   - Players who win by default.
```

File: wiki/Tournaments/oSEAT/5/en.md [hunk 4]
```diff
@@ -0,0 +1,449 @@
...
+   - If there are players who is disqualified from the tournament.
```
Comment (Walavouchey) [inline wiki/Tournaments/oSEAT/5/en.md:415]:
```suggestion
   - If there are players who are disqualified from the tournament.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
may want to apply changes to the rulesets of the previous iterations as well

**Comment by Nivalyx** [comment]
@Walavouchey applied all, thanks for the check as always!

not sure about changing the rulesets of the previous iterations as you suggested tho as i feel like it's out of scope for this pr/it would be more proper to be done in a separate pr instead

File: wiki/Tournaments/oSEAT/5/en.md [hunk 5]
```diff
@@ -0,0 +1,453 @@
...
+| Qualifier ranking | Advance to |
+| :-: | :-- |
+| #1 - #8 | Upper Preliminary group bracket (as **Seed A** players) |
+| #9 - #16 | Upper Preliminary group bracket (as **Seed B** players) |
+| #17 - #24 | Upper Preliminary group bracket (as **Seed C** players) |
+| #25 - #32 | Upper Preliminary group bracket (as **Seed D** players) |
+| #33 - #48 | Lower Preliminary group bracket (as **Seed A** players) |
+| #49 - #64 | Lower Preliminary group bracket (as **Seed B** players) |
+| #65 - #80 | Lower Preliminary group bracket (as **Seed C** players) |
+| #81 - #96 | Lower Preliminary group bracket (as **Seed D** players) |
```
Comment (Walavouchey) [inline wiki/Tournaments/oSEAT/5/en.md:386]: hmm... what do you think

```suggestion
| Qualifier ranking | Preliminary group bracket |
| :-: | :-- |
| #1 - #8 | Upper / **Seed A** |
| #9 - #16 | Upper / **Seed B** |
| #17 - #24 | Upper / **Seed C** |
| #25 - #32 | Upper / **Seed D**) |
| #33 - #48 | Lower / **Seed A** |
| #49 - #64 | Lower / **Seed B** |
| #65 - #80 | Lower / **Seed C** |
| #81 - #96 | Lower / **Seed D** |
```

File: wiki/Tournaments/oSEAT/5/en.md [hunk 6]
```diff
@@ -0,0 +1,453 @@
...
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 12 months of osu!supporter, exclusive [Wrist Labs](https://www.instagram.com/wrist.labs/) goodie pack, unique profile badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 6 months of osu!supporter |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 4 months of osu!supporter |
+| *4th place* | 2 months of osu!supporter |
+| *5th place* | 1 month of osu!supporter |
+
+## Organisation
+
+oSEAT4 is run by various osu! community members predominantly hailing from various South East Asian countries.
```
Comment (Walavouchey) [inline wiki/Tournaments/oSEAT/5/en.md:42]: hang on ya missed this

```suggestion
oSEAT5 is run by various osu! community members predominantly hailing from various South East Asian countries.
```

File: wiki/Tournaments/oSEAT/5/en.md (hunk 5 above)
Comment (Nivalyx) [reply wiki/Tournaments/oSEAT/5/en.md:386]: looks better yeah, i'll go with this

File: wiki/Tournaments/oSEAT/5/en.md [hunk 7]
```diff
@@ -0,0 +1,453 @@
...
+tags:
+  - oSEAT
+  - oSEAT5
+---
+
+# osu! South East Asia Tournament 5
+
+![oSEAT5 logo](img/logo.jpg)
+
+The **osu! South East Asia Tournament 5** (***oSEAT5***) is a double-elimination 1v1 osu! tournament hosted by ![][flag_MY] [Its not Marvin](https://osu.ppy.sh/users/13600612), ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), and ![][flag_FR] [Taevas](https://osu.ppy.sh/users/7276846). The tournament is open to all players from all ten [ASEAN member states](https://asean.org/asean/about-asean/member-states/) (![][flag_BN] Brunei Darussalam, ![][flag_KH] Cambodia, ![][flag_ID] Indonesia, ![][flag_LA] Laos, ![][flag_MY] Malaysia, ![][flag_MM] Myanmar, ![][flag_PH] The Phillipines, ![][flag_SG] Singapore, ![][flag_TH] Thailand, and ![][flag_VN] Vietnam) regardless of rank. It is the 5th instalment of the osu! South East Asia Tournament.
```
Comment (ZeroPyrozen) [inline wiki/Tournaments/oSEAT/5/en.md:11]:
```suggestion
The **osu! South East Asia Tournament 5** (***oSEAT5***) is a double-elimination 1v1 osu! tournament hosted by ![][flag_MY] [Its not Marvin](https://osu.ppy.sh/users/13600612), ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), and ![][flag_FR] [Taevas](https://osu.ppy.sh/users/7276846). The tournament is open to all players from all ten [ASEAN member states](https://asean.org/asean/about-asean/member-states/) (![][flag_BN] Brunei Darussalam, ![][flag_KH] Cambodia, ![][flag_ID] Indonesia, ![][flag_LA] Laos, ![][flag_MY] Malaysia, ![][flag_MM] Myanmar, ![][flag_PH] Philippines, ![][flag_SG] Singapore, ![][flag_TH] Thailand, and ![][flag_VN] Vietnam) regardless of rank. It is the 5th instalment of the osu! South East Asia Tournament.
```

File: wiki/Tournaments/oSEAT/5/en.md [hunk 8]
```diff
@@ -0,0 +1,453 @@
...
+| N | ![][flag_PH] [NathanSS](https://osu.ppy.sh/users/8500628) | ![][flag_ID] [DeZpair](https://osu.ppy.sh/users/7518843) | ![][flag_PH] [ChaeY\_Milk](https://osu.ppy.sh/users/10383440) | ![][flag_JP] (![][flag_PH]) [ZHKPN](https://osu.ppy.sh/users/13042221)² |
+| O | ![][flag_ID] [-TwiHD](https://osu.ppy.sh/users/5470299) | ![][flag_MY] [Houshou Marine](https://osu.ppy.sh/users/6402211) | ![][flag_SG] [D3rpyB01](https://osu.ppy.sh/users/11772019) | ![][flag_SG] [Shleepy](https://osu.ppy.sh/users/8486823) |
+| P | ![][flag_PH] [konawiki](https://osu.ppy.sh/users/4003979) | ![][flag_PH] [Senjuro](https://osu.ppy.sh/users/3003839) | ![][flag_SG] [kirkirs](https://osu.ppy.sh/users/9902622) | ![][flag_SG] [hollowknees](https://osu.ppy.sh/users/15195364) |
+
+¹ *Despite having the ![][flag_US] American flag on their profile, [\_Kolin](https://osu.ppy.sh/users/7249644) is eligible to participate in the tournament as they are in possession of a valid ![][flag_PH] [Phillipine passport](https://en.wikipedia.org/wiki/Philippine_passport).*
```
Comment (ZeroPyrozen) [inline wiki/Tournaments/oSEAT/5/en.md:101]:
```suggestion
¹ *Despite having the ![][flag_US] American flag on their profile, [\_Kolin](https://osu.ppy.sh/users/7249644) is eligible to participate in the tournament as they are in possession of a valid ![][flag_PH] [Philippine passport](https://en.wikipedia.org/wiki/Philippine_passport).*
```

File: wiki/Tournaments/oSEAT/5/en.md [hunk 9]
```diff
@@ -0,0 +1,453 @@
...
+
+² *Despite having the ![][flag_JP] Japanese flag on their profile, [ZHKPN](https://osu.ppy.sh/users/13042221) is eligible to participate in the tournament as they are a citizen of ![][flag_PH] The Phillipines living abroad.*
```
Comment (ZeroPyrozen) [inline wiki/Tournaments/oSEAT/5/en.md:103]:
```suggestion
² *Despite having the ![][flag_JP] Japanese flag on their profile, [ZHKPN](https://osu.ppy.sh/users/13042221) is eligible to participate in the tournament as they are a citizen of ![][flag_PH] Philippines living abroad.*
```

File: wiki/Tournaments/oSEAT/5/en.md [hunk 10]
```diff
@@ -0,0 +1,453 @@
...
+    - Usage of any tools and programs that are against [osu!'s community rules](/wiki/Rules#community-rules) is strictly prohibited and will be straight up reported to the osu! team at will.
+13. Failing to attend in more than two matches without a clear reason of absence will result in a blacklist for the next oSEAT instalment (oSEAT6).
+
+### Tournament registration
+
+1. All interested players are required to register into the tournament individually.
+   - In order to be eligible to play in the tournament, a player must either:
+     - Have the flag of any of the ten [ASEAN member states](https://asean.org/asean/asean-member-states/) (![][flag_BN] Brunei Darussalam, ![][flag_KH] Cambodia, ![][flag_ID] Indonesia, ![][flag_LA] Laos, ![][flag_MY] Malaysia, ![][flag_MM] Myanmar, ![][flag_PH] The Phillipines, ![][flag_SG] Singapore, ![][flag_TH] Thailand, or ![][flag_VN] Vietnam) displayed on their profile, ***or***
```
Comment (ZeroPyrozen) [inline wiki/Tournaments/oSEAT/5/en.md:359]:
```suggestion
     - Have the flag of any of the ten [ASEAN member states](https://asean.org/asean/asean-member-states/) (![][flag_BN] Brunei Darussalam, ![][flag_KH] Cambodia, ![][flag_ID] Indonesia, ![][flag_LA] Laos, ![][flag_MY] Malaysia, ![][flag_MM] Myanmar, ![][flag_PH] Philippines, ![][flag_SG] Singapore, ![][flag_TH] Thailand, or ![][flag_VN] Vietnam) displayed on their profile, ***or***
```

File: wiki/Tournaments/oSEAT/5/en.md (hunk 7 above)
Comment (ZeroPyrozen) [reply wiki/Tournaments/oSEAT/5/en.md:11]: I'm not sure whether it's Philippines or Philippine

**Comment by Nivalyx** [comment]
oh yeah true it should be `The Philippines` instead of `The Phillipines` whoops

fixed!

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**